src/VisualStudio/IntegrationTest/New.IntegrationTests/VisualBasic/BasicIntelliSense.cs

```diff
@@ -31,6 +32,12 @@ public override async Task InitializeAsync()
         {
             await base.InitializeAsync();
 
+            var editorOptionsFactory = await TestServices.Shell.GetComponentModelServiceAsync<IEditorOptionsFactoryService>(HangMitigatingCancellationToken);
+            var options = editorOptionsFactory.GlobalOptions;
+            options.SetOptionValue(DefaultOptions.ResponsiveCompletionOptionId, false);
+            var latencyGuardOptionKey = new EditorOptionKey<bool>("EnableTypingLatencyGuard");
+            options.SetOptionValue(latencyGuardOptionKey, false);
+
```

Comment:
Here's the issue I found locally. Both editor options are enabled here, despite we explicitly disabled them during the call to `base.InitializeAsync()` above ([here](https://github.com/dotnet/roslyn/blob/65fc4bd08b4bc15f6f00b529f47ffaeead03e434/src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/StateResetInProcess.cs#L107)). With this change, I no longer repro the failure locally.

Reply:
@JoshuaBStevens @AmadeusW can you check this? It appears the editor is overwriting our explicit settings somewhere.

Reply:
Filed a bug to track the option issue. Meanwhile, let's merge this to get the test fixed